configure.ac

```
@@ -4750,6 +4750,12 @@ elif test "$disable_gil" = "yes"; then
   AC_MSG_ERROR([--disable-gil requires mimalloc memory allocator (--with-mimalloc).])
 fi
 
+if test "$with_mimalloc" = yes; then
+  if test "$with_tsan" = yes; then
+    AC_DEFINE([MI_TSAN], [1], [Define MI_TSAN for thread sanitizer])
```

Perhaps add a hint that this is a mimalloc constant?

```suggestion
    AC_DEFINE([MI_TSAN], [1], [Define MI_TSAN for mimalloc thread sanitizer support])
```